lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -201,6 +202,10 @@ sub get_dasd_disk_management {
     return Installation::DiskActivation::DASDDiskManagementController->new();
 }
 
+sub get_confirm_installation {
+    return Installation::ConfirmInstallation->new({app => YuiRestClient::get_app()});
```

Comment:
you don't need to pass the app if we get it in that base class as suggested in below comment.

Reply:
I have changed the design quite a bit and now I am  calling LicencePopup from the test layer, so I think now is necessary. Right?